### PR TITLE
fix(react-search): stories tweak

### PR DESCRIPTION
**Pull request by emmayjiang**
Fixes tokens for appearance:
image

Removes stories that do not fit proper use case:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3cbeb7ed15c56a8eee665e5d404c005f01d7c94a'>3cbeb7ed15c56a8eee665e5d404c005f01d7c94a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-search-preview/stories/SearchBox/SearchBoxAppearance.stories.tsx [hunk 1]
```diff
@@ -36,17 +24,11 @@ export const Appearance = () => {
...
         <SearchBox appearance="filled-lighter" />
       </Field>
 
-      <Field
-        className={mergeClasses(styles.fieldWrapper, styles.filledDarker)}
-        label={{ children: 'Filled darker appearance', className: styles.filledDarkerLabel }}
-      >
+      <Field className={styles.fieldWrapper} label="Filled darker appearance">
         <SearchBox appearance="filled-darker" />
       </Field>
```
Comment (sopranopillow) [inline packages/react-components/react-search-preview/stories/SearchBox/SearchBoxAppearance.stories.tsx:33]: Was removing the background intentional? We should have that background as shown in https://react.fluentui.dev/?path=/docs/components-textarea--default#appearance, right?

Comment (emmayjiang) [reply packages/react-components/react-search-preview/stories/SearchBox/SearchBoxAppearance.stories.tsx:33]: ah yes, it was! i forgot to move this PR to draft, it's something i'm looking into with design

**Pull request closed without merging**